extensions/git/src/commands.ts

```diff
@@ -1461,10 +1461,13 @@ export class CommandCenter {
 			}
 		}
 
-		const message = await getCommitMessage();
-
-		if (!message) {
-			return false;
+		let message: string | undefined = '';
+		const amendNoEdit = config.get<boolean>('amendNoEdit');
```

Comment:
Where does this `amendNoEdit` setting come from?

Reply:
does `c4bee30` suffice, or am I missing anything else?